otter_welcome_buddy/graphql/schema/leetcode.gql

```diff
@@ -0,0 +1,357 @@
+  userRating: UserRatingNode
+}
+
+type ArticleNode {
+  id: ID!
+  title: String!
+  content: String
+  contentTypeId: String!
+  paidOnly: Boolean!
+  hasVideoSolution: Boolean!
+  paidOnlyVideo: Boolean!
+  canSeeDetail: Boolean!
+  rating: RatingNode!
+  topic: TopicNode!
+}
+
+type CodeSnippetNode {
+  lang: String!
+  langSlug: String!
+  # String used as template in the code editor
+  code: String!
+}
+
+type QuestionNode {
+  questionId: ID!
+  questionFrontendId: ID!
+  title: String!
+  titleSlug: String!
+  translatedTitle: String
+  categoryTitle: String!
+  isFavor: Boolean!
+  isPaidOnly: Boolean!
+  hide: Boolean!
+  acRate: Float!
+  difficulty: String!
+  # Str in html format to be displayed in the browser
+  content: String!
+  # Encode JSON in format:
+  # {
+  #   totalAccepted: str,
+  #   totalSubmission: str,
+  #    totalAcceptedRaw: int,
+  #    totalSubmissionRaw: int,
+  #    acRate: str
+  # }
+  stats: String!
+  # Hints sorted, the first hint is the easiest one,
+  # is a str in html format
+  hints: [String!]!
+  # SQL schema used in database problems
+  mysqlSchemas: [String!]!
+  # Pandas schema used in database problems
+  dataSchemas: [String!]!
+  # freqBar
+  # eg: "ac", null when no submission
+  status: String
+  hasSolution: Boolean!
+  hasVideoSolution: Boolean!
+  solution: ArticleNode
+  topicTags: [TopicTagNode!]!
+  similarQuestionList: [QuestionNode!]!
+  likes: Int!
+  dislikes: Int!
+  codeSnippets: [CodeSnippetNode!]!
+  canSeeQuestion: Boolean!
+  envInfo: String!
+  hasFrontendPreview: Boolean!
+  frontendPreviews: String!
+  enableDebugger: Boolean!
+  enableRunCode: Boolean!
+  enableSubmit: Boolean!
+  enableTestMode: Boolean!
+  # Examples split by \n, they should match the metadata description
+  exampleTestcaseList: [String!]!
+  # Encode JSON in format:
+  # {
+  #   name: str,
+  #   params: [{
+  #     name: str,
+  #     type: str
+  #   }],
+  #   return: {
+  #     type: str
+  #   }
+  # }
+  metaData: String!
+}
+
+type PagifiedQuestionNode {
+  totalNum: Int!
+  data: [QuestionNode!]!
+}
+
+type CommonTagNode {
+    name: String!
+    slug: String!
+}
+
+type PanelQuestionNode {
+    difficulty: String!
+    id: ID!
+    paidOnly: Boolean!
+    questionFrontendId: String!
+    status: String!
+    title: String!
+    titleSlug: String!
+    score: Int
+    questionNumber: Int
+}
+
+type PanelQuestionListNode {
+  hasViewPermission: Boolean!
+  panelName: String!
+  finishedLength: Int!
+  totalLength: Int!
+  questions: [PanelQuestionNode!]!
+}
+
+input QuestionListFilterInput {
+  listId: String
+  difficulty: String
+  tags: [String!]
+  companies: [String!]
+}
+
+type DailyChallengeNodeV2 {
+  date: String!
+  userStatus: String!
+  link: String!
+  question: QuestionNode!
+}
+
+type CodingChallengeNodeV2 {
+  challenges: [DailyChallengeNodeV2!]!
+  weeklyChallenges: [DailyChallengeNodeV2!]!
+}
+
+type Query {
+  matchedUser(username: String!): UserNode
+  """"""
```

Comment:
Maybe removing these extra `""""""` or why they're needed?

Reply:
Is just to make it more readable